components/table/src/index.js

```diff
@@ -14,70 +14,77 @@ const TableContainer = styled('table', {
   ({ flexibleColumns }) => ({ tableLayout: flexibleColumns ? 'auto' : 'fixed' }),
 );
 
-const TableHeading = styled('th', {
-  forwardProps: ['name'],
-})(({ rowHeading, columnCount }) => ({
+const cellStyles = {
   ':first-child': {
-    padding: '15px 8px 15px 0',
-    width: rowHeading && columnCount < 4 ? '25%' : undefined,
+    padding: '15px 4px 15px 0',
   },
   ':last-child': {
-    padding: '15px 0 15px 8px',
+    padding: '15px 0 15px 4px',
   },
   borderBottom: '1px solid #bfc1c3',
   display: 'table-cell',
   fontSize: '14px',
-  fontWeight: 'bold',
-  textAlign: 'left',
-  verticalAlign: 'baseline',
-}));
+  padding: '15px 4px',
+};
 
 const TableData = styled('td', {
   // use `forwardProps` here as by default emotion doesn't allow setting `name` prop on a `td`
   forwardProps: ['name'],
-})({
-  ':first-child': {
-    padding: '15px 8px 15px 0',
-  },
-  ':last-child': {
-    padding: '15px 0 15px 8px',
-  },
-  borderBottom: '1px solid #bfc1c3',
-  display: 'table-cell',
-  fontSize: '14px',
-  verticalAlign: 'baseline',
-});
+})(
+  cellStyles,
+  ({ verticalAlign }) => ({ verticalAlign }), 
```

Comment:
i'm not entirely sure/convinced about allowing `verticalAlign` to be explicitly set and/or passed through

having a default style of `verticalAlign: 'top'` i think is OK for our current use-cases